.github/README.md

```diff
@@ -0,0 +1,47 @@
+# [Oppia Android](https://www.oppia.org) ![CI Lint and Tests](https://github.com/oppia/oppia-android/workflows/CI%20Lint%20and%20Tests/badge.svg) [![Gitter](https://badges.gitter.im/oppia/oppia-android.svg)](https://gitter.im/oppia/oppia-android?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+Oppia is an online learning tool that enables anyone to easily create and share interactive activities (called 'explorations'). These activities simulate a one-on-one conversation with a tutor, making it possible for students to learn by doing while getting feedback.
+
+The Android app is a new, not-yet-released frontend for Oppia that provides access to Oppia's curated numeracy lessons for users who may not have regular access to internet connectivity, and in a way that can be easily shared across multiple members of the same household.
+
+Oppia Android is written using Kotlin and Java, and leverages Bazel, data-binding, Dagger 2, and AndroidX. See also:
+
+  * [Oppia.org community site](https://www.oppia.org)
+  * [Contributors' wiki](https://github.com/oppia/oppia-android/wiki)
+  * [Developer mailing list](http://groups.google.com/group/oppia-android-dev)
+  * [File an issue](https://github.com/oppia/oppia-android/issues/new/choose)
+
+## Installation
+
+Please refer to the [installation wiki page](https://github.com/oppia/oppia-android/wiki#installation) for full instructions.
+
```

Comment:
As per the brief libraries description, we can add a small list of amazing libraries we are using to support point 1 and 2 in the above comment. 

```
List of amazing libraries we are using
- databinding
- navigation
- com.chaos.view:pinview
- glide
- retrofit2
- de.hdodenhof:circleimageview
- io.fabric.sdk.android:fabric
- kotlin-reflect
- kotlinx-coroutines-android
- androidx.work
- com.caverock:androidsvg
- espresso
- robolectric
- mockito
- google truth
```

Reply:
I think this is a nice idea, though I'm limiting the number of libraries we list to avoid overwhelming the reader. PTAL & let me know what you think. This is definitely a section that we can add onto in the future.

Reply:
Look perfect